backend/app/services/backoffice/csv/base_exporter.rb

```diff
@@ -0,0 +1,35 @@
+require "csv"
+
+module Backoffice
+  module Csv
+    class BaseExporter
+      attr_accessor :query, :columns
+
+      def initialize(query)
+        @query = query
+      end
+
+      def call
+        raise NotImplementedError
+      end
+
+      private
+
+      def generate_csv
```

Comment:
I like this approach , it's very flexible and allows us to add columns to the csv without affecting the backoffice view.